src/Utility.ts

```diff
@@ -48,20 +50,21 @@ export namespace Utility {
         }
     }
 
-    export function getRestartConfig(): boolean {
-        const config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('tomcat');
-        if (config) {
-            return config.get<boolean>(Constants.RESTART_CONFIG_ID);
+    export function browse(localhost: boolean, port: string, subPath?: string, url?: string): void {
```

Comment:
localhost -> isLocalhost

Reply:
will remove the confusing the utility method